metals/src/main/scala/scala/meta/metals/sbtserver/SbtServer.scala

```diff
@@ -113,16 +128,26 @@ object SbtServer extends LazyLogging {
   /**
    * Returns path to project/target/active.json from the base directory of an sbt build.
    */
-  def activeJson(cwd: AbsolutePath): AbsolutePath =
-    cwd.resolve("project").resolve("target").resolve("active.json")
+  object ActiveJson {
+    def apply(): RelativePath =
+      RelativePath("project").resolve("target").resolve("active.json")
+
+    def apply(cwd: AbsolutePath): AbsolutePath =
+      cwd.resolve(ActiveJson())
+
+    def unapply(path: Path): Boolean = {
+      path.endsWith(ActiveJson().toNIO) &&
+      path.toFile.isFile
```

Comment:
Should we also check whether the file exists replacing this with `Files.isRegularFile(path)`?

Reply:
@gabro check 02ee9014c8b5f043df1c924c2639116f489ce049 please. I changed it to matching `RelativePath` + similar changes for `CompilerConfig.File` and `Semanticdbs.File`.